avm/utilities/pipelines/staticValidation/compliance/module.tests.ps1

```diff
@@ -532,6 +532,12 @@ Describe 'Module tests' -Tag 'Module' {
           return
         }
 
+        # workaround for Azure Stack HCI cluster deployment settings resource, where property names have underscores - https://github.com/Azure/Azure-Verified-Modules/issues/1029
+        if ($moduleFolderPath -match 'azure-stack-hci.cluster.deployment-settings') {
```

Comment:
Hey @mbrat2005,
tbanks for the implementation. 💪 

Looks neat - I wonder though if it would make sense to move the if a couple lines down into the `foreach($parameter (...))` block and only except specific parameters for this module folder path, instead of all. What do you think?

Reply:
@AlexanderSehr Sure, that makes sense. I moved the check into the parameter name loop and added the names of the parameters to the check .

Reply:
Great, thanks. The implementation looks good to me. Naturally, a pipeline badge would be great, but as the PR is still in draft I assume it is yet to come.

Independent of that, @eriqua & @jtracey93, would you agree the implementation addresses the issue in a good way?

Reply:
> Great, thanks. The implementation looks good to me. Naturally, a pipeline badge would be great, but as the PR is still in draft I assume it is yet to come.
> 
> Independent of that, @eriqua & @jtracey93, would you agree the implementation addresses the issue in a good way?

@AlexanderSehr - I don't have a pipeline badge due to some PSRule bicep expansion business, but added a screenshot of the testing output with updated condition

Reply:
Looks good, thanks @mbrat2005 💪